ocs_ci/ocs/ui/base_ui.py

```diff
@@ -138,10 +182,11 @@ def login_ui():
 
         # headless browsers are web browsers without a GUI
         headless = ocsci_config.UI_SELENIUM.get("headless")
-        if headless:
+        if not headless:
             chrome_options.add_argument("--headless")
 
-        chrome_browser_type = ocsci_config.UI_SELENIUM.get("chrome_type")
+        # chrome_browser_type = ocsci_config.UI_SELENIUM.get("chrome_type")
+        chrome_browser_type = "google-chrome"
```

Comment:
isn't this some debugging leftovers?

Reply:
I changed this for debugging the code.. I will change it..